src/vanclj/members.clj

```diff
@@ -45,4 +45,9 @@
 
    {:name   "David Dossot"
     :github "https://github.com/ddossot"
-    :twitter "https://twitter.com/ddossot"}])
+    :twitter "https://twitter.com/ddossot"}
+   
+   {:name   "Fenton Travers"
+    :github "https://github.com/ftravers"}
```

Comment:
Same question for you as for Joel :smile: Do you have a pic for yourself or a link to gravatar to use?

You can put it under the `:avatar` key